### DISCUSSION TITLE
is is possible to override the admin module razor pages to control the layout?

### DISCUSSION THREAD

**Discussion by skuflow** [oqtane/oqtane.framework#2971] [General]
I don't want to deviate from the core functionality but the admin layout is lacking an eye catching pop which my clients want.  Is there a way to override the admin razor page layouts without basically forking the code?  I don't see any id or classes in container divs so it's not really possible to push stuff around the page via css.

**Comment by sbwalker** [comment]
You can create a custom theme based on Bootstrap 5 - the Admin modules all use standard Bootstrap classes for styling. If you want to completely customize the UI to meet the needs of your personal clients, you can fork the modules and create your own.

**Comment by skuflow** [comment]
I don't want to fork it because that was one of my major issues with DNN.  You all really should use id's or classes in your code so that we can easily grab divs and style them and move them around the page as needed.

**Comment by sbwalker** [comment]
@skuflow I am assuming what you are asking for is a div wrapper around the admin components which has a unique id attribute so that you can target them with CSS - correct?
